jira/client.py

```diff
@@ -366,6 +366,10 @@ class JIRA:
             # 'Expires': 'Thu, 01 Jan 1970 00:00:00 GMT'
             "X-Atlassian-Token": "no-check",
         },
+        "default_batch_size": {
+            Resource: 100,
+            Issue: 500,
```

Comment:
```suggestion
```

Let's not make this the default for now, obviously feel free to increase this for your own system